unit-tests/run-unit-tests.py

```diff
@@ -40,6 +40,18 @@ def usage():
     print( '                         with what tags it has' )
     print( '        --no-exceptions  do not load the LibCI/exceptions.specs file' )
     print( '        --context        The context to use for test configuration' )
+    print()
+    print( 'Examples:' )
+    print( 'Running: python <path-to-exe-dir> run-unit-tests.py -r cpp -t log' )

```

Comment:
What's `<path-to-exe-dir>`?

Reply:
the path to the directory holding the exe files for the tests

Reply:
Isn't it misplaced?

Reply:
You are correct

Reply:
I'd make this example last and actually give an example of such a directory:
`python run-unit-tests.py -r name -t log ~/my-build-directory`